hieradata_aws/class/training/db_admin.yaml

```diff
@@ -0,0 +1,23 @@
+govuk_env_sync::tasks:
+  "pull_content_data_admin_integration_daily":
+    ensure: "present"
+    hour: "0"
+    minute: "18"
+    action: "pull"
+    dbms: "postgresql"
+    storagebackend: "s3"
+    database: "content_data_admin_integration"
+    temppath: "/tmp/content_data_admin_integration_pull"
+    url: "govuk-integration-database-backups"
+    path: "postgresql-backend"
+  "push_content_data_admin_integration_daily":
+    ensure: "present"
+    hour: "1"
+    minute: "18"
+    action: "push"
+    dbms: "postgresql"
+    storagebackend: "s3"
+    database: "content_data_admin_integration"
```

Comment:
All db names remain (...)_production in all envs

Reply:
Have made the change, could you review again? Thanks!